OpenRA.Game/ModData.cs

```diff
@@ -41,11 +41,12 @@ public ModData(string mod, bool useLoadScreen = false)
 		{
 			Languages = new string[0];
 			Manifest = new Manifest(mod);
+			ModFiles.LoadFromManifest(Manifest);
 
 			// Allow mods to load types from the core Game assembly, and any additional assemblies they specify.
-			var assemblies =
-				new[] { typeof(Game).Assembly }.Concat(
-					Manifest.Assemblies.Select(path => Assembly.LoadFrom(Platform.ResolvePath(path))));
+			var assemblies = new[] { typeof(Game).Assembly }
+					.Concat(Manifest.Assemblies.Select(path => Assembly.Load(ModFiles.Open(path).ReadAllBytes())))
```

Comment:
Note that this will temporarily break the ability to load assemblies from mods in the support dir, but, like the other cases, this will be restored in #10703.